prowler/providers/azure/azure_provider_new.py

```diff
@@ -0,0 +1,270 @@
+import sys
+from os import getenv
+from typing import Any, Optional
+
+from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential
+from azure.mgmt.subscription import SubscriptionClient
+from colorama import Fore, Style
+from msgraph.core import GraphClient
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.providers.azure.lib.regions.regions import get_regions_config
+from prowler.providers.common.provider import CloudProvider
+
+
+class Azure_Identity_Info(BaseModel):
```

Comment:
I think we should remove `_` from the class names since is not the convention in Python.

Reply:
Classes names now match camelcase structure